packages/cli/src/lib/defaults/build-strategies/wasm/javascript/vm/VERSION

```diff
@@ -1 +1 @@
-0.1.0
\ No newline at end of file
+0.1.1
```

Comment:
NOTE: to release this, you need to add `| /workflows/cd-containers` to your PR's title: https://github.com/polywrap/cli/blob/origin-dev/.github/workflows/cd-containers.yaml#L12

Also, be sure to update the VM image version within the strategy: https://github.com/polywrap/cli/blob/fe8b36f676e11aaac958a75475b05961681700a7/packages/cli/src/lib/build-strategies/strategies/DockerVMStrategy.ts#L53